### DISCUSSION TITLE
可以期待一下 Cascadia Next SC, TC, and JP

### DISCUSSION THREAD

**Discussion by weicong** [subframe7536/maple-font#237] [General]
![image](https://github.com/user-attachments/assets/9907386f-9fa5-4d45-98be-8c86d807ca3a)

**Comment by subframe7536** [comment]
看了一下，实现思路和更纱黑体是一致的：中文使用思源黑体，并且通过减少英文字符宽度达到2：1等宽

**Comment by rocascent** [comment]
英文字符有点太窄了。
不过感觉中文方面用黑体确实会好看一些。

**Comment by rocascent** [comment]
不知道鸿蒙黑体能不能用来开发字体啊，个人感觉很好看

**Reply by subframe7536** [reply]
没办法

目前市面上许可证允许二开的中文字体只有思源黑体，国内厂商做的字体想都别想

最开始v5就是用汉仪正圆，我也是别人提醒之后才发现的
